rllib/algorithms/algorithm.py

```diff
@@ -2154,6 +2154,9 @@ def load_checkpoint(self, checkpoint_dir: str) -> None:
             learner_state_dir = os.path.join(checkpoint_dir, "learner")
             self.learner_group.load_state(learner_state_dir)
 
+        # Call the `on_checkpoint_loaded` callback.
+        self.callbacks.on_checkpoint_loaded(algorithm=self)
```

Comment:
@sven1977 is there any reason why `checkpoint_dir` is not passed into `on_checkpoint_loaded()`? I think this would be very useful, as in my current use case, I want to save artifacts based on the respective checkpoint that has been loaded.